utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -34,6 +34,10 @@ BeforeDiscovery {
     # Import any helper function used in this test script
     Import-Module (Join-Path $PSScriptRoot 'helper' 'helper.psm1') -Force
 
+    # Load used functions
+    . (Join-Path $repoRootPath 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModuleTargetVersion.ps1')
```

Comment:
Beware that it can be very problematic to use this function. We used to leverage in the readme generation and if I recall correctly it worked fine until we actually merged the feature into `main` after which it immediately started failing the static tests. It has been a long time, but I think the issue was that running the function in main after the merge would, naturally, again give you the 'next' version after the current - even though in reality the current version is already the 'latest'.  
We never got around to fix the original logic (which still exists in some branch) - but I guess it would require some branch checks etc.